src/main/java/com/metamx/collections/bitmap/WrappedRoaringBitmap.java

```diff
@@ -146,7 +147,6 @@ public void serialize(ByteBuffer buffer)
     if (compressRunOnSerialization) {
       bitmap.runOptimize();
     }
-    buffer.putInt(getSizeInBytes());
```

Comment:
Can't backward compatibility be enabled with a flag? Because saving 4 bytes per bitmap can make a huge difference in some cases where you have many tiny bitmaps (10 bytes each), I see that this can happen with Druid...